src/controllers/scripting/controllerscriptenginebase.cpp

```diff
@@ -100,25 +100,19 @@ void ControllerScriptEngineBase::showScriptExceptionDialog(
     QString backtrace = evaluationResult.property("stack").toString();
     QString filename = evaluationResult.property("fileName").toString();
 
-    QString errorText;
     if (filename.isEmpty()) {
-        errorText = QString("Uncaught exception at line %1 in passed code.")
-                            .arg(line);
-    } else {
-        errorText = QString("Uncaught exception at line %1 in file %2.")
-                            .arg(line, filename);
+        filename = QStringLiteral("<passed code>");
```

Comment:
???

Reply:
As this may trigger multiple times I suggest to put it into an anonymous namespace as a constant.

Reply:
> ???

I'm sorry if I did something wrong or questionable. If I knew what you were questioning, I wouldn't have done it wrong in the first place, so please explain what you think is wrong with this code. Thanks

Reply:
Sorry, I got confused by the old code without taking the time to understand the surrounding context. This looks okay.

Reply:
Yeah, I essentially wrote new code and deleted the old one but since it was so similar git just created a very confusing diff. It's all good.